lib/parseConfig/index.js

```diff
@@ -46,7 +50,26 @@ module.exports = function (config) {
 			advanced: operation.inputSchema.advanced || [],
 		};
 
-		return message;
+		if (_.isObject(operation.subOperations)) {	
+			var subOperations = operation.subOperations;
+
+			if (_.isObject(subOperations.destroy)) {
+				message.destroy = getModel(subOperations.destroy);
+			}
+			if (_.isObject(subOperations.dynamic_output)) {
+				message.dynamicOutput = getModel(subOperations.dynamic_output);
+			}
+			if (_.isObject(subOperations.request)) {
+				message.request = getRequestModel(subOperations.request);
```

Comment:
`getRequestModel=` and `getResponseModel` both look completely unnecessary? Why not just do
`normalizeModelParameter(ubOperations.request.model)` and `response` respectively?